libs/ui-toolkit/src/primitives/typography.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import classNames from 'classnames';
+import type { ComponentStory } from '@storybook/react';
+type Args = { type: string; alternatives: string; isAlpha: boolean };
+const sizeArr = [
+  'text-3xl',
+  'text-2xl',
+  'text-xl',
+  'text-lg',
+  'text-lg',
+  'text-base',
+  'text-sm',
+  'text-xs',
+];
+const TextSample = ({ alternatives, isAlpha, type }: Args) => {
+  return (
+    <div className="m-4 flex h-16">
+      <div className="text-small flex flex-col mr-16 justify-end w-32">
+        {type}:
+      </div>
+      <div
+        className={classNames(
+          'flex-grow flex flex-col justify-end text-left items-start',
+          { 'font-alpha': isAlpha },
+          [alternatives, type]
+        )}
+      >
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit
```

Comment:
suggestion (non blocking): Maybe add a section with ligatures as we'll start introducing that in some select places

Reply:
done